### ISSUE TITLE
Enable Numlock by default

### ISSUE THREAD

**Issue by abernardinello** [CIRB/devbox#32]
At this time the only way i found to enable num lock at login is to install numlockx and call it in sessionCommands. Have you a better way to do this, and can you add it by default?

λ ~ → sudo nvim /etc/nixos/configuration.nix

..........

      sessionCommands = ''
        ${pkgs.xlibs.xsetroot}/bin/xsetroot -cursor_name left_ptr
        ${pkgs.feh}/bin/feh --bg-scale "$HOME/.wallpaper.jpg"
        /home/vagrant/.nix-profile/bin/numlockx on # activate numlock @ login
      '';
...........
:wq

λ ~ → nix-env -i numlockx
λ ~ → sudo nixos-rebuild switch

**Comment by mlefevre** [comment]
I tried to set the Alexis solution through local configuration file 
```
# Local customization that won't be overridden by vagrant provision
{ config, lib, pkgs, ... }:
{
  environment.systemPackages = with pkgs; [
    (eclipses.eclipseWithPlugins {
       eclipse = eclipses.eclipse-sdk-452;
       plugins = [ ];
     })
    firefox
    gitAndTools.tig
    # geany
    numlockx
  ];
  services.xserver = {
    displayManager = {
      sessionCommands = ''
        numlockx on
      '';
    };
  };
}
```
I copied the local-configuration.nix file in /vagrant (=<host-devbox-directory>)
Then I destroyed my box and rebuild it anew to be sure that the local configuration truly worked.
Indeed the numlockx package is installed :) but it seems that the ```numlockx on @ login``` doesn't work.

Once the box is up, numlockx on|off work like a charm, so the error must be in the services.xserver section I'm using in this local configuration. Does someone has an idea ?

**Comment by pradermecker** [comment]
@mlefevre Would you mind trying to replace `numlockx on` with `${pkgs.numlockx}/bin/numlockx on`. You can also remove numlockx from the list of the package as it will be pick up automatically.

PS: To get a quicker response please ping me on the message or use slack.

**Comment by pradermecker** [comment]
@abernardinello @mlefevre I was planning to apply the change into `configuration.nix` for v3. I can make the change in v2 if you tell me it works as expected.

**Comment by pradermecker** [comment]
fix in f48ffe6f9
